sdks/python/apache_beam/transforms/sideinputs_test.py

```diff
@@ -309,6 +316,66 @@ def test_flattened_side_input(self):
     assert_that(results, equal_to(['a', 'b']))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_multi_triggered_gbk_side_input(self):
+    """Test a GBK sideinput, with multiple triggering."""
+    options = PipelineOptions()
+    options.view_as(StandardOptions).streaming = True
```

Comment:
A cleaner way of setting streaming in pipeline options is: `options = StandardOptions(streaming=True)`.